### PR TITLE
create eks cluster

### PR DESCRIPTION
**Pull request by gerryhzgtest2021**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.2.0"
+
+  azs             = ["us-east-1a", "us-east-1b"]
+  name            = "main"
+  private_subnets = ["10.0.2.0/24", "10.0.3.0/24"]
+  public_subnets  = ["10.0.0.0/24", "10.0.1.0/24"]
+  cidr            = "10.0.0.0/16"
+  #vpc
```
Comment (vladimirmukhin) [inline main.tf]: Is this comment really needed? For me, it's clear from the code that this is VPC-related configs

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  enable_dns_hostnames = true
+  enable_classiclink   = false
+  #NAT Gateway
```
Comment (vladimirmukhin) [inline main.tf]: Same regarding comment here

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.eks.certificate_authority[0].data)
+  token                  = data.aws_eks_cluster_auth.eks.token
+}
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.2.0"
+
+  azs             = ["us-east-1a", "us-east-1b"]
+  name            = "main"
```
Comment (vladimirmukhin) [inline main.tf]: Naming should be consistent between VPC and EKS 
Use ${local.env} here as well

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (gerryhzgtest2021) [reply main.tf]: Sure. I will delete it.

File: main.tf (hunk 2 above)
Comment (gerryhzgtest2021) [reply main.tf]: deleted.

File: main.tf (hunk 3 above)
Comment (gerryhzgtest2021) [reply main.tf]: changed.

Comment (gerryhzgtest2021) [reply main.tf]: Thanks a lot!

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**